pixel-saver@deadalnix.me/convenience.js

```diff
@@ -0,0 +1,96 @@
+/* -*- mode:js; indent-tabs-mode:true; tab-width: 4 -*- */
```

Comment:
Standard mode line. Do you prefer a different tab width?

I don't think this must be mentioned in the README. Better changing the width according to your preferences.